rasa/core/test.py

```diff
@@ -960,11 +993,19 @@ async def test(
         _log_stories(
             story_evaluation.failed_stories,
             os.path.join(out_directory, FAILED_STORIES_FILE),
+            "None of the test stories failed - all good!",
         )
     if successes and out_directory:
         _log_stories(
             story_evaluation.successful_stories,
             os.path.join(out_directory, SUCCESSFUL_STORIES_FILE),
+            "None of the test stories succeeded :(",
```

Comment:
I might be confused now - but shouldn't this comment be in the `if errors` branch earlier - and then the `None of the test stories failed - all good!" comment should be in the `if successes` branch? Basically, should they be swapped?

Reply:
No, this message is what gets printed when there are no successful stories — `successes` is basically a CLI flag that specifies if we need to create a `SUCCESSFUL_STORIES_FILE`